home/tests/test_models.py

```diff
@@ -797,7 +796,6 @@ def test_template_create_with_example_values(self):
         )
         wat.save()
         wat.save_revision()
-        wat.submit_whatsapp_template(None)
 
         request = responses.calls[0].request
```

Comment:
Do we have a test here for what happens when the call to whatsapp fails? If not, then we should probably add one